bin/GameIndex.dbf

```diff
@@ -2863,8 +2865,10 @@ Region = NTSC-K
 Serial = SCKA-20032
 Name   = Syphon Filter - The Omega Virus
 Region = NTSC-K
-EETimingHack = 1 // Random hangs.
+EETimingHack = 1 // Fixes Random hangs.
```

Comment:
If you like you can change Random not to use capital letter, your choice since it's a very small nitpick.

Reply:
Done